test/common_framework.py

```diff
@@ -114,18 +114,34 @@ def setUp(self):
         copytree("commands", join(WORKDIR, "dosemu"), symlinks=True)
 
         # create minimal startup files
+        self.setUpDosAutoexec()
+        self.setUpDosConfig()
+        self.setUpDosVersion()
+
+    def setUpDosConfig(self):
         mkfile(self.confsys, """\
+SWITCHES=/F\r
```

Comment:
Mm, and no way to copy it instead
of hard-coding inside the script?

Reply:
This is only for the non FDPP and non FreeDOS cases. In the generic config.sys script there are lines that I cannot guarantee will be okay on all other DOSes e.g.
`device=c:\dos\himem.sys`
`install=d:\dosemu\emufs.com`
`shell=command.com /e:1024 /p:autoemu.bat`
or
`shell=c:\command.com /e:1024 /p /k autoemu.bat `

So not great, but trying to pick those lines out and replace is likely to be just as horrid.

Reply:
Yes, I agree.
The tests have failed however.

Reply:
Mmm, some odd random FDPP crashes I see. I wonder if it's because I'm now loading `emufs.com`, wasn't there some recent issue with that?
~~~
                                                                              

dosemu XMS 3.0 & UMB support enabled

dosemu EMS driver rev 0.8 installed.

EMUFS host file and print access available

dosemu CDROM driver installed (V0.2)

Kernel: allocated 115 Diskbuffers = 62560 Bytes in HMA

emufs: redirector enabled

Process 0 starting: G:\command.com /e:384 /k %FDPP_AUTOEXEC%

ERROR: MCB corruption, good:02b8:0000 bad:9f84:0000

ERROR: Please install gdb!

fdpp crashed.

ERROR: fdpp: abort at memmgr.cc:166

ERROR: Please install gdb!

Press any key to exit.

~~~

Reply:
Any reproducer?
Bad news. :(

Reply:
Is it 32 or 64 bits?

Reply:
Random crashes in that they occur on different tests each run, but so far always FDPP. Both Travis and my local machine are 64bit Bionic Ubuntu. It has never happened on my local machine, though that may be just good(?) luck. I'm trying again after temporarily REMming out the `install dosemu\emufs.com`

Reply:
The first run completed okay, I'll rerun for a total of 5 times, or until I get a failure.

Reply:
So I ran it 5 times with no failure. I'm thinking that for now I should just drop the commit 
7d529d2 until the problem can be reproduced locally?

Reply:
But it would be good to keep it in a
separate PR, to just see if its still crashing.

Reply:
Okay, will do.